images/sds-local-volume-scheduler-extender/src/cmd/main.go

```diff
@@ -54,13 +54,14 @@ const (
 )
 
 type Config struct {
-	ListenAddr             string  `json:"listen"`
-	DefaultDivisor         float64 `json:"default-divisor"`
-	LogLevel               string  `json:"log-level"`
-	CacheSize              int     `json:"cache-size"`
-	HealthProbeBindAddress string  `json:"health-probe-bind-address"`
-	CertFile               string  `json:"cert-file"`
-	KeyFile                string  `json:"key-file"`
+	ListenAddr             string        `json:"listen"`
+	DefaultDivisor         float64       `json:"default-divisor"`
+	LogLevel               string        `json:"log-level"`
+	CacheSize              int           `json:"cache-size"`
+	HealthProbeBindAddress string        `json:"health-probe-bind-address"`
+	CertFile               string        `json:"cert-file"`
+	KeyFile                string        `json:"key-file"`
+	PVCExpiredDurationSec  time.Duration `json:"pvc-expired-duration-sec"`
```

Comment:
Does this struct will ever be marshalled to json? If not, let's remove `json` tags. If somehow it will, there's a bug:

```go
	result := &Config{
		PVCExpiredDurationSec: 124, // 124 seconds, right?
	}
	if b, err := json.Marshal(result); err != nil {
		panic(err)
	} else if err := json.Unmarshal(b, result); err != nil {
		panic(err)
	}

	fmt.Printf("%s\n", result.PVCExpiredDurationSec)
```

Output:
```
124ns
```

The value has been transformed after marshalling/unmarshalling.

Solution is to use `int` instead of `time.Duration`